includes/class-simple-local-avatars.php

```diff
@@ -406,9 +406,12 @@ class="button item-delete submitdelete deletion"
 						$profileuser->simple_local_avatar_rating = 'G';
 					}
 
-					foreach ( $this->avatar_ratings as $key => $rating ) :
-						echo "\n\t<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( $key ) . "' " . checked( $profileuser->simple_local_avatar_rating, $key, false ) . "/> $rating</label><br />";
-					endforeach;
+                                        // These are the same as $this->avatar_ratings but in order for string translations to appear we have to output them like the following
+                                        // We also use the default text domain here
+                                        echo "<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( 'G' ) . "' " . checked( $profileuser->simple_local_avatar_rating, 'G', false ) . "/>" . __( 'G &#8212; Suitable for all audiences' ) . "</label><br />";
+                                        echo "<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( 'PG' ) . "' " . checked( $profileuser->simple_local_avatar_rating, 'PG', false ) . "/>" . __( 'PG &#8212; Possibly offensive, usually for audiences 13 and above' ) . "</label><br />";
+                                        echo "<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( 'R' ) . "' " . checked( $profileuser->simple_local_avatar_rating, 'R', false ) . "/>" . __( 'R &#8212; Intended for adult audiences above 17' ) . "</label><br />";
+                                        echo "<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( 'X' ) . "' " . checked( $profileuser->simple_local_avatar_rating, 'X', false ) . "/>" . __( 'X &#8212; Even more mature than above' ) . "</label><br />";
```

Comment:
Moving it to a separate function and updating `$this->avatar_ratings`, and calling just before the `foreach` starts - would be a better improvement. Then no need to change anything in the `foreach` at all.

The function would just have the same array from `__constructor`, overwriting the existing `$this->avatar_ratings`.

https://github.com/10up/simple-local-avatars/blob/86ce042ae571583986d0c439675fc3f9be5be04a/includes/class-simple-local-avatars.php#L15-L20

Also, using `esc_html` over `$ratings` would be an extra improvement.

https://github.com/10up/simple-local-avatars/blob/86ce042ae571583986d0c439675fc3f9be5be04a/includes/class-simple-local-avatars.php#L410